docs/reference/content/getting-started/quick-tour.md

```diff
@@ -0,0 +1,381 @@
+{{% /note %}}
+
+### Find All Documents in a Collection
+
+To retrieve all the documents in the collection, we will use the
+`find()` method. The `find()` method returns a `FindObservable` instance that
+provides a fluent interface for chaining or controlling find operations. 
+The following code retrieves all documents in the collection and prints them out
+(101 documents). Using the `printResults()` implicit we block until the observer is completed and then print each result:
+```scala
+collection.find().printResults()
+```
+
+## Get A Single Document with a Query Filter
+
+We can create a filter to pass to the find() method to get a subset of
+the documents in our collection. For example, if we wanted to find the
+document for which the value of the "i" field is 71, we would do the
+following:
+
+```scala
+import org.mongodb.scala.model.Filters._
+
+collection.find(equal("i", 71)).first().printHeadResult()
+```
+
+will eventually print just one document:
+
+```json
+{ "_id" : { "$oid" : "5515836e58c7b4fbc756320b" }, "i" : 71 }
+```
+
+{{% note %}}
+Use the [Filters]({{< apiref "org.mongodb.scala.model.Filters$">}}), [Sorts]({{< apiref "org.mongodb.scala.model.Sorts$">}}), 
+[Projections]({{< apiref "org.mongodb.scala.model.Projections$">}}) and [Updates]({{< apiref "org.mongodb.scala.model.Updates$">}}) 
+helpers for simple and concise ways of building up queries and updates
+{{% /note %}}
+
+## Get a Set of Documents with a Query
+
+We can use the query to get a set of documents from our collection. For
+example, if we wanted to get all documents where `"i" > 50`, we could
+write:
+
+```scala
+collection.find(gt("i", 50)).printResults()
+```
+which should print the documents where `i > 50`.
+
+We could also get a range, say `50 < i <= 100`:
+
+```scala
+collection.find(and(gt("i", 50), lte("i", 100))).printResults()
+```
+
+## Sorting documents
+
+We can also use the [Sorts]({{< apiref "org.mongodb.scala.model.Sorts$">}}) helpers to sort documents.
+We add a sort to a find query by calling the `sort()` method on a `FindObservable`.  Below we use the [`exists()`]({{< apiref "org.mongodb.scala.model.Filters$@exists(fieldName:String,exists:Boolean):org.bson.conversions.Bson">}}) helper and use the sort
+[`descending("i")`]({{< apiref "org.mongodb.scala.model.Sorts$@descending(fieldNames:String*):org.bson.conversions.Bson">}}) helper to sort our documents:
+
+```scala
+collection.find(exists("i")).sort(descending("i")).first().printHeadResult()
+```
+
+## Projecting fields
+
+Sometimes we don't need all the data contained in a document. The [Projections]({{< apiref "org.mongodb.scala.model.Projections$">}}) 
+helpers can be used to build the projection parameter for the find operation and limit the fields returned.  
+Below we'll sort the collection, exclude the `_id` field and output the first matching document:
+
+```scala
+collection.find().projection(excludeId).first().printHeadResult()
```

Comment:
Good catch - I think creating a document is a side effect so the overload should be defined with params. 

Scala has opinions about empty parens: http://docs.scala-lang.org/style/naming-conventions.html#parentheses